packages/state/daos_hidden_from_search.json

```
@@ -0,0 +1,10 @@
+[
+  "juno1hgu46yeryyhcmjfr30h0607m4a48sd9penvt4sw85nftn5l3lgaqqlc99h",
```

```suggestion
// Duplicate Juno SubDAOs from testing / etc.
  "juno1hgu46yeryyhcmjfr30h0607m4a48sd9penvt4sw85nftn5l3lgaqqlc99h",
```

Like commenting a little rational for why DAOs are hidden. Potentially useful later if we ever have to revise the list or justify why certain DAOs are included here.